### ISSUE TITLE
<Toaster /> duration prop not changing duration of the notification

### ISSUE THREAD

**Issue by clarkdowding** [timolins/react-hot-toast#10]
Giving your nice react-hot-toast npm a run in a project I'm working on. We have a simplistic alerts system and just today started looking at libs to upgrade our UI. Trying yours out after reading a blurb from a @tylermcginnis newsletter.

We literally copied this code from your <Toaster /> API page and tweaked the duration really long to 30 seconds (30000) while testing some other things but the duration doesn't seem to be honored:
```javascript
<Toaster
  position="top-center"
  reverseOrder={false}
  toastOptions={{
    // Define default options
    style: {
      margin: '40px',
      background: '#363636',
      color: '#fff',
      zIndex: 1,
    },
    duration: 5000,
    // Default options for specific types
    success: {
      duration: 3000,
      theme: {
        primary: 'green',
        secondary: 'black',
      },
    },
  }}
/>
```

We will keep playing around with it and let you know if we determine it is something in our code.

**Comment by timolins** [comment]
Woa, thanks for reporting this! Just checked and it seems like the default duration from `toastOptions` is not getting passed on. Though it should work with `toast("", {duration: 50000})`. Will try to get a fix out as soon as possible.

**Comment by clarkdowding** [comment]
Just tested passing the prop at the toast() invocation as you suggested and indeed it worked great. Tnx.

**Comment by timolins** [comment]
Just published `v1.0.1` which addresses this issue. Let me know if that worked for you!

**Comment by clarkdowding** [comment]
Confirming that ```v1.0.1``` as delivered through ```npm update``` this evening fixes Issue #10. Thanks!

**Comment by tylermcginnis** [comment]
Chiming in here to say I love this lib @timolins. Just added it to our project (replacing react-toastify) and it's a joy to work with. Great job.
